Tasks/Common/webdeployment-common/xmlvariablesubstitutionutility.ts

```diff
@@ -53,6 +122,10 @@ export async function substituteXmlVariables(configFile, tags, variableMap){
                     if(xmlNode.name == "configSections") {
```

Comment:
Can we add user facing logs (not debug logs) like "configSections replacement complete", "parameter file replacement complete" etc?